language/Dutch/strings.po

```diff
@@ -56,7 +56,7 @@ msgstr "Bezels"
 
 #: src/frontend/mame/ui/videoopt.cpp:220
 msgid "CPanels"
-msgstr "BPanelen"
+msgstr "CPanelen"
```

Comment:
Ah, I know why it was "B":
msgid "Controls Panels"
msgstr "Bedieningspanelen"

Reply:
Changed back